readme.md

```diff
@@ -45,3 +45,26 @@ Copyright Quentin Lebastard and Stefan Kerkmann 2021.
 * Splithand detection - ok
 * Bootloader bodge - ok, can enter reliably everytime
 * Matrix - ok
+
+## BOM
+
+* Blackpill qty. 2
+    * [AliExpress](https://www.aliexpress.com/item/1005001456186625.html)
+* STM M95 SPI EEPROM (SOP-8 package size) qty. 2 | **Note:** make sure it's no more than 64kbit as QMk does not currently support more than that.
+    * [LCSC Electronics](https://lcsc.com/product-detail/EEPROM_STMicroelectronics-M95640-WMN6TP_C140882.html)
+* 5.1k 1206 Resistor qty. 4
+    * [LCSC Electronics](https://lcsc.com/product-detail/Chip-Resistor-Surface-Mount_UNI-ROYAL-Uniroyal-Elec-1206W4F5101T5E_C26033.html)
+* TRRS Jack 3.5mm
+    * [LCSC Electronics](https://lcsc.com/product-detail/Pre-ordered-Connectors_XKB-Connectivity-PJ-320E_C2884941.html)
+* Push button 5.1mm x 1.5mm
+    * [LCSC Electronics](https://lcsc.com/product-detail/Tactile-Switches_XKB-Connectivity-TS-1187A-B-A-B_C318884.html)
+
+## JLCPCB Order Form (non default values)
+
+* Different Design: 2
+* PCB Height: 1mm
+
+## Recommended way to attach MCU to adapter
+
+![Header pins long side on bottom](https://cdn.discordapp.com/attachments/747849854276993114/915438746885816401/STM32F411CEU6_WeAct_Black_Pill_V2.png "bottom view of WeAct_Black_Pill")
```

Comment:
I think the picture is a bit misleading, as the MCU will be mounted flipped so the bottom is actually upside down.

Reply:
oh ok. Do you have a better picture (or better yet a rendering?)

Reply:
would this one be better?
https://cdn.discordapp.com/attachments/747850923023532073/917080283470262292/YDJeRnB-WmwzC29bB0-DFyqS0bw50P-dWDTpCJ_VeNJU6AFi0Cd2YPfCxlG4ZC24qSIUgrxeY2SqNYOBj9AjxDTzOZSGVp5PYvswRtTRKmT6I8PHPesjp8GcBe7GgcOz4FociG-_ST2pCwWIUDUHhSAQGFm50UViV4P7pItD_OBN_bQ0HBUbJ3WU-Oz9lr7biuISDuU_EBOzYPIBNygqnm3tUYzpScuLZzDeZDKdSb6KK_nFvDO_z_WIr5U03MFT1mPGZdNkEBX79G5o5_3yKE4uKm1fy5IhMoevBQppwjXil2UEoJ2bYBu1xJPmckNVj6lIyWO2Gq5qVVJ0clMITZysOZz7uf5ep086iT0VnurOK-IWEBglufBhdpOvbv9KoBGky6LOEV_hgE1BaJWvfOJAbr0cHQaaFzvvDEhSTnF-yGUnIEhWwl-BHmFBdxLxZ5-ENgsxnDAfqvF1rb8MR39nZcgViq9kDO5t2g-8pF_9mazi0uXdEGEFyd4V..png

Reply:
Yes, this one hits the spot! Could you add it as a picture to the PR? Don't know how long Discord keeps their stuff around.